### ISSUE TITLE
RocketChat container tag 3.18.6 is running 4.7.2 

### ISSUE THREAD

**Issue by xoxys** [RocketChat/Rocket.Chat#25659]
The RC container tagged with 3.18.6 is running the wrong **major** version:

```
docker ps
CONTAINER ID   IMAGE                           COMMAND                  CREATED          STATUS                  PORTS                                                                      NAMES
fd1785336e46   rocketchat/rocket.chat:3.18.6   "docker-entrypoint.s…"   12 minutes ago   Up 12 minutes           3000/tcp                                                                   rocketchat
```

```
# docker logs fd1785336e46
[...]
+--------------------------------------------------+
|                  SERVER RUNNING                  |
+--------------------------------------------------+
|                                                  |
|  Rocket.Chat Version: 4.7.2                      |
|       NodeJS Version: 14.18.3 - x64              |
|      MongoDB Version: 4.0.28                     |
|       MongoDB Engine: unknown                    |
|             Platform: linux                      |
|         Process Port: 3000                       |
|             Site URL: https://chat.example.com  |
|     ReplicaSet OpLog: Enabled                    |
|          Commit Hash: 1688633945                 |
|        Commit Branch: HEAD                       |
|                                                  |
+--------------------------------------------------+
```

That's a massive issue as it has migrated our instances to the new major version including breaking changes.....

**Comment by reetp** [comment]
Can confirm this. Using fixed Docker version, not latest.

Going to be a lot of angry people shortly. Sheeesh.

**Comment by xoxys** [comment]
And I guess no simple downgrade route....

**Comment by ali-alhaidary** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/25660

**Comment by ali-alhaidary** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/25657

**Comment by debdutdeb** [comment]
im checking the digests on docker hub and this dowsn't seem to be the case :/

**Comment by sparse91** [comment]
Same issue with direct download link: https://releases.rocket.chat/3.18.6/download

**Comment by casalsgh** [comment]
People, 3.18.6 was removed from docker. 
3.18.7 is available on docker hub -> https://hub.docker.com/layers/rocket.chat/rocketchat/rocket.chat/3.18.7/images/sha256-ac495e672234ba1e625acd8af8a23ba85f08a87a23dbfc299815597595f2bf64?context=explore

Aware of the problematic download link and working to fix it.

**Comment by casalsgh** [comment]
> And I guess no easy downgrade route....

Unfortunately No. 
Suggestion from reetp might work.

**Comment by casalsgh** [comment]
> > And I guess no easy downgrade route....
> 
> Unlikely beyond dump your Mongo DB and restore from your backup, which you took just before you upgraded..... didn't you?
> 
> Bad enough tagging latest wrong but this is ridiculous.
> 
> Absolutely howling QA.

Sorry for the inconvenience caused. We are working to fix and improve processes so that it doesn't happen again.

**Comment by casalsgh** [comment]
> Same issue with direct download link: https://releases.rocket.chat/3.18.6/download

Please use link: https://releases.rocket.chat/3.18.7/download

**Comment by geekgonecrazy** [comment]
A few things:

1. I've fixed the download link for: https://releases.rocket.chat/3.18.6/download
2. I've corrected the 3.18.6 docker image. 
3. This fix and the fix against these different versions was QA'd before released and passed. 

This was an out of the ordinary situation.  We don't typically patch this many versions as the same time.  We chose to patch more versions then actually required to try and maximize protection for your sake.

Unfortunately because of so many builds happening at the same time.. yes we did run into some issues with the CI as well as with managing that many patch release branches..

Just so you know when a patch is developed for the latest version and we back port it.  We can't just always apply the commit.  It often requires basically rewriting the patch again.  Unfortunately in this case even though QA'd while managing that many branches and releases at the same time.  Mistakes happened.

In any case.. Sorry :pray:  And we will improve how we do multiple rapid patch releases in the future.  Thanks for your patience!
